api/filters.go

```diff
@@ -530,6 +531,58 @@ func (api *FilterAPI) removeFilterBlueprintDimensionOption(w http.ResponseWriter
 	log.Info("delete filtered blueprint", log.Data{"filter_blueprint_id": filterID, "dimension": name})
 }
 
+func (api *FilterAPI) getFilterOutputPreview(w http.ResponseWriter, r *http.Request) {
+	vars := mux.Vars(r)
+	filterID := vars["filter_output_id"]
+	requestedLimit := r.URL.Query().Get("limit")
+	var limit int64 = 20
+	if requestedLimit != "" {
+		limit, err := strconv.ParseInt(requestedLimit, 10, 64)
```

Comment:
Consider making the limit variable type `int`, and using `strconv.Atoi` to convert it

Reply:
done